src/org/usfirst/frc/team467/robot/ElevatorHeightSensor.java

```diff
@@ -0,0 +1,59 @@
+package org.usfirst.frc.team467.robot;
+import edu.wpi.first.wpilibj.AnalogInput;
+
+public class ElevatorHeightSensor {
+	
+	AnalogInput sensor;
+	static ElevatorHeightSensor instance;
+	public double rotations;
+	public double height;
+	public double inchConvert = 16.9;  // number of ticks per inch
+	public double feetConvert = 202.8; //number of ticks per foot
+	public double rotationConvert = 253; //number of ticks per rotation
```

Comment:
Rename this to describe exactly what it is. Call it ticksPerRotation. Then when you use it the code will look like:

`sensor.getValue() / ticksPerRotation`

which will read more easily.